### DISCUSSION TITLE
Conversion rate per landing page / referral source

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1115] [Anything Else]
"I can see I have 20 people who clicked my CTA button on the pricing page, but I can't find a percentage anywhere? I want to know which of my articles and landing pages are converting better so I can make more posts, not only based on absolute numbers."

Right now we show this for individual sources/pages when you drill down by goal + source/page but adding them all on the main view when you're viewing a drill down of a particular custom event/goal would make it easier to discover the most efficient sources / pages. Basically 10 conversions out of 50 Google visitors is better than 11 conversions out of 1000 visitors from Reddit which currently it takes a bit of extra work to figure out.

Could be nice to add conversion rate from the top chart either under the "Details" view or next to the absolute numbers (perhaps like we show % for browsers/OS/devices on the main view).

(feedback via email)

**Update:** This is now live

**Comment by mlent** [comment]
Big +1 from me :)

**Comment by benjaminhouy** [comment]
I would love to see this as well :)

**Comment by metmarkosaric** [comment]
this is now live thanks to the contribution by @ro-savage! conversion rate (CR) for all the metrics is shown both in the main view and in the details view when filtered by a goal. 

we plan to do a couple of tweaks over the next few days to:

* rename "visitors" to "conversions" when filtered by goal as we're showing the number of converted visitors rather than the number of total visitors
* add the number of visitors to the details view when filtered by goal so it's easier to understand what's actually performing best (for instance some source may have a very high CR but very low number of actual visitors so would be nice with a quick overview that helps you understand that at a glance)

**Reply by mlent** [reply]
Love this, Marko! Just tried it out, so useful. A quick question/request - would it be possible to make the conversion rate column sortable when looking at the page details after selecting a Goal?

Right now it's automatically sorted by number of visitors, but it would be super handy to be able to sort by conversion rate when you're dealing with a lot of pages.

Thanks @ro-savage for the feature :)

**Reply by metmarkosaric** [reply]
glad to hear that @mlent, thanks! and yes, we plan to make all the columns in the "details" views sortable by whichever metric you click on. cannot make any promise on when but there's a thread on this idea here https://github.com/plausible/analytics/discussions/458

**Comment by benjaminhouy** [comment]
This is awesome! Thanks @ro-savage and @metmarkosaric ! :)

**Reply by metmarkosaric** [reply]
thank you @benjaminhouy!
